README.md

```diff
@@ -15,6 +15,67 @@
 3. Blazing Fast: Tachyon's foremost requirement is speed, and not just any speed, but blazing speed! This entails Tachyon delivering exceptional performance on both CPU and GPU platforms.
 4. GPU Interoperability: Tachyon's code is designed to be compatible with both CPU and GPU in the majority of scenarios.
 
+## List of Features
+
+List of symbols:
+
+- :heavy_check_mark: means the feature is currently supported.
+- 🏗️ means that the feature is partially implemented or is under active construction.
+- :x: means that the feature is not currently supported.
```

Comment:
```suggestion
- 🏗️ means the feature is partially implemented or is under active construction.
- :x: means the feature is not currently supported.
```
How about matching the rhymes like the above line like this?

Reply:
oops i also recommended a way to fix this